novuhq/website
# add community page

This pull request brings community page.

Steps to check:

- open [preview](https://deploy-preview-166--novu-website.netlify.app/community/)
- make sure that everything looks and works as expected
